### DISCUSSION TITLE
serverpod path issue on macos

### DISCUSSION THREAD

**Discussion by awaisajmal22** [serverpod/serverpod#2025] [Q&A] [answered]
i recently install server pod when i run the command serverpod it gives an error .zsh connot find the command i can i solve thi issue

**Comment by Frank1016** [comment, marked as answer]
It looks like the serverpod command isn't recognized in your shell, which usually indicates that the Serverpod executable isn't in your system's PATH. After installing Serverpod, you need to make sure the Dart pub bin directory has been added to your system's PATH. For a typical setup, you might need to add the following line to your .zshrc file, which is located in your home directory (~/.zshrc):

`export PATH="$PATH":"$HOME/.pub-cache/bin"`

**Comment by awaisajmal22** [comment]
i got it thank you so much

**Comment by awaisajmal22** [comment]
thank you the issue has been resolved
